client/src/selectors/index.js

```diff
@@ -60,15 +73,15 @@ export const paginatedNoticesSelector = createSelector(
   paginationSelector,
   (dateSorted, nameSorted, orderBy, page) => {
     const notices = orderBy === 'title' ? nameSorted : dateSorted
-    return chunk(notices, PAGINATION_CHUNK_SIZE)[page - 1]
+    return chunk(notices, PAGINATION_CHUNK_SIZE)[page - 1] || []
   }
 )
 
 // not the most elegant, but presently we need the whole list
 // sorted by date anyway
 export const newestBulletinDateSelector = createSelector(
   dateSortedNoticesSelector,
-  (notices) => notices[0].bulletin_date
+  (notices) => notices[0] ? notices[0].bulletin_date : ''
 )
 
 // will make more sense once we allow searching on notices
```

Comment:
This comment seems realted :slightly_smiling_face: Shouldn't you also change `noticesLengthSelector` (it's used in Pagination of Notices)?